code/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+@import url(https://fonts.googleapis.com/css?family=Noto+Sans);
+
+/*** Hero styling ***/
+
+h1 {
+  text-align: center;
+}
+
+body {
+  margin: 0;
+}
+
+main {
+  background-image: url("./pizza.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  background-color: rgba(255, 255, 255, 0.8);
+  width: 50%;
+  font-family: "Playfair Display", serif;
+  margin: 50px;
+  padding: 20px;
+  font-size: 15px;
+}
+
+.welcome-message-container {
+  margin-top: 0px;
+  font-family: "Cabin", Sans-Serif;
+  font-size: 15px;
+}
+
+.question-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  font-family: "Cabin", Sans-Serif;
+  font-size: 15px;
+}
+
+.question-container input {
+  width: 250px;
+  padding: 5px;
+  display: flex;
+  font-family: "Cabin", Sans-Serif;
+  font-size: 15px;
+  background-color: rgb(255, 255, 255);
+}
+
+.submit-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+}
+
+.form-submit {
+  width: 150px;
+  padding: 15px;
+  font-size: 15px;
+  cursor: pointer;
+  border: 1px solid 0.5;
```

Comment:
`0.5` of what - `px`, `meters`, `pizzas`? ;)
Btw, most browsers doesn't detect values lower than `1px` since it's the smallest unit, so we should use only fixed numbers here